instances/latest/terminologies/molecularEntity/4-(2-Hydroxyethyl)-1-piperazineEthanesulfonicAcid.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/4-(2-Hydroxyethyl)-1-piperazineEthanesulfonicAcid",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/MolecularEntity",
+  "definition": "HEPES (4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid) is a zwitterionic sulfonic acid buffering agent; one of the twenty Good's buffers. [adapted from wikipedia (https://en.wikipedia.org/wiki/HEPES)]",
```

Comment:
This PR was just to synchronise with controlledTerms. I'm happy to make changes to `@id`s  and names (e.g. to fix weird capitalization in PubChem) since it will cause problems in the KG if we change them later, but everything else (definitions, descriptions, synonyms) I suggest we tackle in a separate PR.